qa/tasks/cephfs/test_volumes.py

```diff
@@ -56,8 +59,15 @@ def _wait_for_clone_to_fail(self, clone, clone_group=None, timo=120):
     def _check_clone_canceled(self, clone, clone_group=None):
         self.__check_clone_state("canceled", clone, clone_group, timo=1)
 
-    def _verify_clone_attrs(self, subvolume, clone, source_group=None, clone_group=None):
-        path1 = self._get_subvolume_path(self.volname, subvolume, group_name=source_group)
+    def _get_subvolume_snapshot_path(self, snapshot, subvol_path):
```

Comment:
When [this](https://github.com/ceph/ceph/pull/35756) PR is completed, version based handling of path to the snapshot would get taken care of.